lib/attributesParser.js

```diff
@@ -4,13 +4,36 @@
 */
```

Comment:
Please move this into its own folder && file => `./lib/attrs/srcset.js` and `require('./attrs/srcset.js')` it